src/locales/en.json

```diff
@@ -214,6 +214,7 @@
       "cozy": {
         "accounts": "Login details",
         "apps": "List of installed applications",
+        "apps.suggestions": "Suggestions of applications",
```

Comment:
Nit: I would say "Application suggestions" sounds more natural, but I don't have the whole context.

Reply:
Thanks, fixed.